platform/features/layout/src/FixedController.js

```diff
@@ -374,7 +374,7 @@ define(
         FixedController.prototype.updateView = function (telemetryObject, datum) {
             var metadata = this.openmct.telemetry.getMetadata(telemetryObject);
             var telemetryKeyToDisplay = this.chooseTelemetryKeyToDisplay(metadata);
-            var formattedTelemetryValue = this.getFormattedTelemetryValueForKey(telemetryKeyToDisplay, datum, metadata);
```

Comment:
This circumvents `chooseTelemetryKeyToDisplay` which causes a regression (#1740).  Instead of circumventing `chooseTelemetryKeyToDisplay` you should find & fix the bug in `chooseTelemetryKeyToDisplay`.